src/Controllers/ElementalAreaController.php

```diff
@@ -45,132 +47,121 @@ public function getClientConfig()
         $clientConfig = parent::getClientConfig();
         $clientConfig['form']['elementForm'] = [
             'schemaUrl' => $this->Link('schema/elementForm'),
-            'saveUrl' => $this->Link('api/saveForm'),
-            'saveMethod' => 'post',
-            'payloadFormat' => 'json',
-            'formNameTemplate' => sprintf(static::FORM_NAME_TEMPLATE ?? '', '{id}'),
+            // 'saveUrl' => $this->Link('elementForm'),
+            'saveMethod' => 'post', // and remove this?
+            // 'payloadFormat' => 'json',
+            'formNameTemplate' => sprintf(static::FORM_NAME_TEMPLATE, '{id}'),
         ];
-
         // Configuration that is available per element type
         $clientConfig['elementTypes'] = ElementTypeRegistry::generate()->getDefinitions();
-
         return $clientConfig;
     }
 
     /**
-     * @param HTTPRequest|null $request
-     * @return Form
-     * @throws HTTPResponse_Exception
+     * Used for both:
+     * - GET requests to get the FormSchema via getElementForm() called from LeftAndMain::schema()
+     * - POST Requests to save the Form. Will be handled by to FormRequestHandler::httpSubmission()
+     * /admin/linkfield/elementForm/<ElementID>
+     *
+     * @param HTTPRequest $request - deprecated 5.3.0 Will be removed without equivalent functionality to replace it
      */
-    public function elementForm(HTTPRequest $request = null)
+    public function elementForm(HTTPRequest $request = null): Form
     {
-        // Get ID either from posted back value, or url parameter
-        if (!$request) {
-            $this->jsonError(400);
-            return null;
-        }
-        $id = $request->param('ID');
+        $request = $request ?: $this->getRequest();
+        $id = $request->param('ItemID');
+
+        // Respect previous behaviour
```

Comment:
I'm probably being overly backwards compatible here by not changing HTTP status codes in a minor release. Especially considering I've completely disabled apiSaveForm(). Feel free to recommend that we don't respect previous http codes and change it to something better